### PR TITLE
Agent release 2.2.4

### PR DESCRIPTION
**Pull request by alesnovak-s1**

### CODE REVIEW

**Review by jmakar-scalyr — CHANGES_REQUESTED** [review]
These changes are ok, there just should be some more work done first

The last time I did a release I used this as a guideline (I don't know if Arthur pointed you to better/newer documentation): https://www.notion.so/sentinelone/Release-management-6c14b353ea08417fab69c58cba4c2790

Specifically I saw that the 2.2.4 CHANGELOG entry needed a "name" (and possibly more details) and more importantly that the 2.2.4 image tags have not yet been uploaded to https://hub.docker.com/r/scalyr

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
-## 2.2.4 "TBD" - Jun 22, 2023
+## 2.2.4 "Trillian" - Jun 22, 2023
 <!---
-Packaged by Arthur Kamalov <arthurk@sentinelone.com> on Jun 22, 2023 00:00 -0800
+Packaged by Ales Novak <ales.novak@sentinelone.com> on Jul 14, 2023 00:00 -0800
```
Comment (jmakar-scalyr) [inline CHANGELOG.md:6]: Should this be Aug 14?

**Review by jmakar-scalyr — APPROVED** [review]
Looks good
Btw I think we want to use `2.2.4` as the image tag not `v2.2.4` (ref: https://hub.docker.com/r/scalyr/scalyr-agent-docker/tags?page=1&name=2.2)

File: CHANGELOG.md (hunk 1 above)
Comment (alesnovak-s1) [reply CHANGELOG.md]: Thanks for catching that.

**Pull request merged**